packages/app/src/styles/theme/_apply-colors-dark.scss

```diff
@@ -373,6 +375,26 @@
         box-shadow: none !important;
       }
     }
+
+    // bookmarks
+    .grw-folder-tree-container {
+      .grw-drop-item-area , .grw-foldertree-item-container {
+        .grw-accept-drop-item {
+          border-color: hsl.lighten(var(--bgcolor-sidebar-context), 30%) !important;
+        }
+      }
+    }
+    .grw-bookmarks-list {
+      @include override-list-group-item-for-pagetree(
+        var(--color-sidebar-context),
+        hsl.lighten(var(--bgcolor-sidebar-context),8%),
+        hsl.lighten(var(--bgcolor-sidebar-context),15%),
+        hsl.darken(var(--color-sidebar-context),15%),
+        hsl.darken(var(--color-sidebar-context),10%),
+        hsl.lighten(var(--bgcolor-sidebar-context),18%),
+        hsl.lighten(var(--bgcolor-sidebar-context),24%)
+      );
+    }
```

Comment:
Is this `grw-bookmarks-list` class used?

Reply:
This class is no longer used, so I removed it